setup.py

```diff
@@ -7,7 +7,7 @@
 with open('README.md') as readme_file:
     readme = readme_file.read()
 
-requirements = ['webexteamssdk==1.6.1', 'coloredlogs', 'websockets==11.0.3', 'backoff']
+requirements = ['webexpythonsdk', 'coloredlogs', 'websockets==11.0.3', 'backoff']
```

Comment:
Please pin to latest version

Reply:
Pointed webexpythonsdk to the latest version (2.0.2)